src/ConcreteOperations/issubset.jl

```diff
@@ -526,6 +526,46 @@ function ⊆(x::Interval, y::Interval, witness::Bool=false)
     return x.dat ⊆ y.dat
 end
 
+function ⊆(x::Interval, U::UnionSet, witness::Bool=false)
+    @assert dim(U) == 1 "an interval is incompatible with a set of dimension $(dim(U))"
+    V = [U.X, U.Y]
+    if !(V isa AbstractVector{<:Interval})
+        V = [convert(Interval, Y) for Y in V]
+    end
+    return _issubset_interval(x, V, witness)
+end
+
+function ⊆(x::Interval, U::UnionSetArray, witness::Bool=false)
+    @assert dim(U) == 1 "an interval is incompatible with a set of dimension $(dim(U))"
+    V = array(U)
+    if !(V isa AbstractVector{<:Interval})
+        V = [convert(Interval, Y) for Y in V]
+    end
+    return _issubset_interval(x, array(U), witness)
+end
+
+function _issubset_interval(X::Interval{N}, intervals, witness) where {N}
```

Comment:
```suggestion
function _issubset_interval!(X::Interval{N}, intervals, witness) where {N}
```
(since `intervals` may be changed)